### PR TITLE
feat: columns in composed chart

### PR DESCRIPTION
**Pull request by simcha90**

### CODE REVIEW

File: plugins/plugin-chart-composed/src/plugin/configs/orderBy.ts [hunk 1]
```diff
@@ -57,17 +59,24 @@ const getOrderByRow = (source: string, name: string, title: string, index?: numb
...
 ];
 
-orderByMetric.push(getOrderByRow('metrics', 'metric', t('"metric"')));
-
 for (let i = 0; i < MAX_FORM_CONTROLS; i++) {
   orderByGroupBy.push(getOrderByRow('groupby', 'group_by', t('"group by"'), i));
 }
 
+for (let i = 0; i < MAX_FORM_CONTROLS; i++) {
+  orderByGroupBy.push(getOrderByRow('metrics', 'metric', t('"metric"'), i));
```
Comment (amitmiran137) [inline plugins/plugin-chart-composed/src/plugin/configs/orderBy.ts:76]: shouldn't that be orderByMetric?

File: plugins/plugin-chart-composed/src/plugin/controlPanel.ts [hunk 2]
```diff
@@ -198,7 +238,16 @@ const config: ControlPanelConfig = {
...
       label: t('Query'),
       expanded: true,
-      controlSetRows: [[groupBy], [metrics], ['columns'], ['adhoc_filters'], ['row_limit', null]],
+      controlSetRows: [
+        [queryMode],
+        [xAxisColumn],
+        [yAxisColumn],
+        [groupBy],
+        [metrics],
+        ['columns'],
```
Comment (amitmiran137) [inline plugins/plugin-chart-composed/src/plugin/controlPanel.ts:247]: why do we need that one?

File: plugins/plugin-chart-composed/src/plugin/utils.ts [hunk 3]
```diff
@@ -33,6 +39,9 @@ export type FormData = {
   layout: Layout;
   colorScheme: string;
   minBarWidth: string;
+  queryMode: QueryMode;
+  xAxisColumn: string;
+  yAxisColumn: string;
```
Comment (amitmiran137) [inline plugins/plugin-chart-composed/src/plugin/utils.ts:44]: xColumn
yColumn

**Review by amitmiran137 — COMMENTED** [review]
a few comments

File: plugins/plugin-chart-composed/src/plugin/controlPanel.ts (hunk 2 above)
Comment (simcha90) [reply plugins/plugin-chart-composed/src/plugin/controlPanel.ts:247]: it's breakdowns

**Pull request merged**